sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADITHelper.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.test.aad.selenium;
+
+import com.azure.spring.utils.AzureCloudUrls;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.azure.spring.test.EnvironmentVariable.AAD_SINGLE_TENANT_CLIENT_ID;
+import static com.azure.spring.test.EnvironmentVariable.AAD_SINGLE_TENANT_CLIENT_SECRET;
+import static com.azure.spring.test.EnvironmentVariable.AAD_TENANT_ID_1;
+import static com.azure.spring.test.EnvironmentVariable.AZURE_CLOUD_TYPE;
+
+public class AADITHelper {
+
+    public static Map<String, String> createDefaultProperties() {
```

Comment:
Why extract this to a new class?

Reply:
There are many references, so I extracted them here.

Reply:
Can we keep original `AADSeleniumITHelper`?